etl-replicator/migrations/20250616122411_create_replication_state_table.sql

```diff
@@ -11,10 +11,10 @@ create table
     etl.replication_state (
         pipeline_id bigint not null,
         table_id oid not null,
-        state table_state not null,
-        -- TODO: should use the pg_lsn type here but simplifying because sqlx doen't yet support a PgLsn type
-        -- and we'd have to implement it manually (and the Encode/Decode) traits to make it work. So taking
-        -- the simpler approach of using text type for now.
+        state etl.table_state not null,
```

Comment:
Even with the `set search_path = 'etl';` it was giving me issues in which the `table_state` type was not found. Made it explicit here.

Reply:
This should be tested by running migrations on an older database since changing an existing migration file will change the hash. Not sure how sqlx will behave in this case though.

Reply:
Ah damn, good point. It would throw an error indeed. I might just skip this change then. I don't want to introduce more problems.